js/src/launchAsync.ts

```diff
@@ -231,7 +266,7 @@ export function launchAsync(token: string, subdomain: string, content: Content,
             });
         }
 
-        const domain = options.customDomain ? options.customDomain : `https://${subdomain}.cognitiveservices.azure.com/immersivereader/webapp/v1.0/`;
+        const domain = options.customDomain ? options.customDomain : 'https://learningtools.onenote.com/learningtoolsapp/cognitive/';
```

Comment:
What does this do? If a subdomain is mistakenly passed by a 1st party app would it result in a 401?

Reply:
I'm not 100% sure about the logic since this was copied from the 0.0.3 sdk, but subdomain is passed into the web app (see line 170) so shouldn't be an issue here

---
In reply to: [508876023](https://github.com/microsoft/immersive-reader-sdk/pull/201#discussion_r508876023) [](ancestors = 508876023)